firmware/application/apps/ui_recon.cpp

```diff
@@ -539,10 +528,21 @@ ReconView::ReconView(NavigationView& nav)
         nav_.push<AnalogAudioView>();
     };
 
-    button_looking_glass.on_select = [this](Button&) {
-        nav_.pop();
-        nav_.push<GlassView>();
+    button_loop_config.on_select = [this](Button&) {
+        if (continuous) {
+            continuous = false;
+            button_loop_config.set_style(&Styles::white);
+        } else {
+            continuous = true;
+            button_loop_config.set_style(&Styles::green);
+        }
+        persistent_memory::set_recon_continuous(continuous);
     };
+    if (continuous) {
```

Comment:
nit: consider moving into a little helper function and all these new lines can be removed.
could be like
```
set_continuous(bool cont) {
  continuous = cont;
  button_loop_config.set_style(const ? &Styles::green : &Styles::white);
}

Reply:
done